### ISSUE TITLE
hot reloading doesn't work in docker container

### ISSUE THREAD

**Issue by nickdex** [vuetifyjs/pwa#12]
I tried out other templates from official vue-cli and hot-reloading works on them. I tested both webpack and PWA templates. I'm also getting this exception on both local and docker 
```
(node:2502) UnhandledPromiseRejectionWarning: Error: Exited with code 1
    at ChildProcess.cp.once.code (/mnt/c/Users/nikwa/Documents/Projects/Web/Pasteit-Anywhere/node_modules/opn/index.js:84:13)
    at Object.onceWrapper (events.js:255:19)
    at ChildProcess.emit (events.js:160:13)
    at maybeClose (internal/child_process.js:943:16)
    at Socket.stream.socket.on (internal/child_process.js:363:11)
    at Socket.emit (events.js:160:13)
    at Pipe._handle.close [as _onclose] (net.js:559:12)
(node:2502) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:2502) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

**Comment by nekosaur** [comment]
You need to give us a bit more information, otherwise this will be very hard for us to troubleshoot. At the very least you need to give step by step instructions on how you've setup your docker environment and vuetify template so that we can try to replicate the issue.

**Comment by nickdex** [comment]
Here's the docker command I used
```
docker run --name=test1 -v c:/Users/user/Documents/Projects/Web/TestProject:/home/node --user=node -it node:latest bash
```
Then after running `cd /home/node` and installing vue-cli (locally) with yarn I ran `./node_module/vue-cli/bin/vue init vuetifyjs/pwa`
It asked me bunch of questions and after setting all up, I ran `yarn install` to install all deps. Installation was successful since I can see the template website loaded into the browser however when I make any changes it won't hot-reload.
I used `yarn run dev` to start the server. Hot reloading works fine when I use it on my local machine.
Btw any changes I make in my editor on local machine do get reflected in container. I just checked it again to be sure so, volume mapping is also working as expected
